lib/msal-angular/package.json

```diff
@@ -92,9 +92,9 @@
     "clean": "shx rm -rf dist docs",
     "doc": "typedoc --out ./docs ./src/**/* --gitRevision dev",
     "build:modules": "tsc && tsc -m es6 --outDir lib-es6",
-    "build": "npm run clean && npm run doc && npm run build:modules && webpack",
+    "build": "npm run clean && npm run build:modules && webpack",
```

Comment:
Why are we removing the docs here?

Reply:
Please see my comment summary on the PR above. I removed the docs temporarily as the doc script was failing on travis.

Reply:
It used to work. Do you know what is the issue. May be we should fix the issue instead of removing it,